pkgs/applications/networking/maestral-qt/default.nix

```diff
@@ -28,6 +29,7 @@ python3.pkgs.buildPythonApplication rec {
   ];
 
   buildInputs = [
+    qtwayland
     qt6.qtbase
     qt6.qtsvg  # Needed for the systray icon
```

Comment:
```suggestion
    qtbase
    qtsvg  # Needed for the systray icon
```

Reply:
@NickCao Ah, that means that those inputs are implicitly controlled by the variant of `callPackage` (`Qt5` vs `Qt6`)? And that would also mean that the derivation remains the same for both Qt versions but is built as requested by `callPackage`?

Reply:
There are some minor differences between qt5 and qt6 modules, but mostly the derivation can be shared. And yes all the modules are brought into the scope by qt{5,6}.callPackage.

Reply:
Oh and you need to fix the parameters too, replacing `qt6` with the actually used components, squash the commits after everything.